model_compression_toolkit/core/common/mixed_precision/mixed_precision_search_manager.py

```diff
@@ -66,7 +67,10 @@ def __init__(self,
         self.layer_to_bitwidth_mapping = self.get_search_space()
         self.compute_metric_fn = self.get_sensitivity_metric()
 
-        self.compute_ru_functions = ru_functions
+        self.cuts = calc_graph_cuts(self.original_graph)
```

Comment:
Maybe add a todo to find a better solution for running the cuts computation? 
The goal is that we won't need to run the max cut computation in case we are not in activation (or total) constrained mixed precision.